### ISSUE TITLE
image HDRI 

### ISSUE THREAD

**Issue by disainerus** [lllyasviel/Fooocus#1648]
Hello. I want to create the interior of rooms.
Can your project make (or will it make in the future) a panoramic image HDRI from the middle of such a room for use in 3D program (for example, 3D max or Blender or Three JS).

**Comment by mashb1t** [comment]
Currently Fooocus can not create this kind of images.
I am not aware of any possibility to also make them consistent as SDXL has been trainned on specific 1024*1024 and derivates in aspect ratios with the same sum of pixels, which means it will start to repeat elements if the image differs too much from the trained resolutions.
You can find more information here: https://github.com/lllyasviel/Fooocus/pull/1617

Fooocus might have a feature in the future when a new model supporting higher resolution is published. For now, you can keep extending images with outpaint to one direction (left or right), but consistency is not guaranteed.

Does this answer your question?

**Comment by disainerus** [comment]
Yes, thank you

**Comment by disainerus** [comment]
I'm sorry, maybe I didn't ask the question quite right. I am interested in a panoramic image (as in the attached file). Currently, this is done by the resource https://skybox.blockadelabs.com/ But it is more intended for outdoor use.
![павильон 5](https://github.com/lllyasviel/Fooocus/assets/93859368/7b24ba5b-a9c2-4b6b-b123-8e65676c024a)

**Comment by mashb1t** [comment]
> Currently Fooocus can not create this kind of images.

Panoramic image generation is not (yet) possible in Fooocus.

**Comment by disainerus** [comment]
Ok, thank you

**Comment by barepixels** [comment]
Yes, you can do it in Fooocus (SDXL). The trick is to render 1024x512 then upscale and use this Lora https://huggingface.co/artificialguybr/360Redmond

Here is a tutorial
https://www.reddit.com/r/StableDiffusion/comments/16csnfr/workflow_creating_a_360_panorama_image_or_video/

But I advice you to do it with SD1.5 (a111) because of this handy extension (have not been updated in 2 years)
https://github.com/tjm35/asymmetric-tiling-sd-webui

What it does is auto blend the far **left edge** to the far **right edge** seemless and avoid this problem if choose SDXL route

![seems](https://github.com/lllyasviel/Fooocus/assets/92839033/0a3f1c3a-1054-48a5-8f2d-3dc7ec8301ad)

**Comment by disainerus** [comment]
Ok, thank you very much, I will study and try
